feature/bgp/policybase/otg_tests/aspath_and_community_test/README.md

```diff
@@ -0,0 +1,128 @@
+# RT-2.4: BGP Policy AS Path Set and Community Set
+
+## Summary
+
+BGP policy configuration for AS Paths and Community Sets
+
+## Procedure
+
+* RT-2.4.1 - Test setup
+  * Generate config for 2 DUT ports, with DUT port 1 eBGP session to ATE port 1.
+
+  * Generate config for ATE 2 ports, with ATE port 1 eBGP session to DUT port 1.
+
+  * Configure ATE port 1 to advertise ipv4 and ipv6 prefixes using the following aspaths and communities:
+    * prefix-set-1 with as path `[100, 200, 300]` and communities `[100:1, 200:2, 300:3]`
+    * prefix-set-2 with as path `[100, 400, 300]` and communities `[101:1]`
+    * prefix-set-3 with as path `[109]` and communities `[109:1]`
+    * prefix-set-4 with as path `[200]` and communities `[200:1]`
+    * prefix-set-5 with as path `[300]` and communities `[100:1]`
+
+  * Establish eBGP sessions between ATE port-1 and DUT port-1
+  * Generate traffic from ATE port-2 to all prefixes
+  * Validate that traffic is received on ATE port-1 for all prefixes
+
+* RT-2.4.1 - Validate single routing-policy containing as-path-set and ext-community-set
```

Comment:
I do not see that this test plan covers ext-community-set

Reply:
I take this as we require extended community feature?  I will make that a separate test in a separate PR if required.

Reply:
Discussed offline, will make a separate test in future PR for extended community.